source/type/pb_converter.cpp

```diff
@@ -620,18 +620,10 @@ void PbCreator::Filler::fill_pb_object(const nt::Line* l, pbnavitia::Line* line)
          * We could have link the LineSection impact with the line, but that would change the code and
          * the behavior too much.
          * */
-        std::set<boost::shared_ptr<nt::disruption::Impact>> added_impact;
         auto fill_line_section_message = [&](const nt::VehicleJourney& vj) {
-            for(const auto& impact: vj.meta_vj->impacted_by) {
```

Comment:
`impacted_by` should probably be renamed `impacted_and_modified_by` to avoid this kind of mistake

Reply:
I've changed `impacted_by` by `modified_by`